File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int pp=0;
        int x=1;
        for(int i=0; i<num_list.length; i++){
            pp+=num_list[i];
            x*=num_list[i];
        }
        if(pp*pp>x){
            answer+=1;
        }
        return answer;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.py
def solution(num_list):
    answer = 0
    pp = 0
    x = 1
    for i in num_list :
        pp+=i
        x*=i
    if (pp*pp>x) : #합의 제곱이 더 크다면?
        answer+=1
    return answer